holoviews/operation/datashader.py

```diff
@@ -1584,7 +1585,9 @@ class spread(SpreadingOperation):
         Number of pixels to spread on all sides.""")
 
     def _apply_spreading(self, array):
-        return tf.spread(array, px=self.p.px, how=self.p.how, shape=self.p.shape)
+        replace_none_how = ds_version <= '0.11.1' and (self.p.how is None)
+        how = 'source' if replace_none_how else self.p.how
```

Comment:
That's some pretty convoluted logic. What about:

```
def _calculate_how_from_default(how):
    """Handle changes in values accepted for `how` across Datashader versions"""
    if ds_version <= '0.11.1' and (how is None):
         how = 'source'
    return how

how = _calculate_how_from_default(self.p.how)
```

Reply:
Personally, I would rather not introduce such a trivial function. Happy to let @philippjfr be the tie-breaker...

Reply:
Don't want to just change the default depending on the version?

Reply:
That would also work and might be simpler. I'll try that idea, thanks!